README.md

```diff
@@ -153,14 +159,14 @@ For now, the logger provides two output type: `pretty` and `json`, default is `j
       
     logger.setNamespaces('namespace:*');
     logger.setLevel('debug');
-    //logger.setOutput('json');
+    //logger.setOutput(logger.outputs.json);
```

Comment:
I think it's weird to use this here in a commented instruction while we never mentioned the availability of it in the doc before.

Reply:
Ok So i removed the // so it's not anymore a comment